src/Asset/EntrypointLookupCollection.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony WebpackEncoreBundle package.
+ * (c) Fabien Potencier <fabien@symfony.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\WebpackEncoreBundle\Asset;
+
+use Symfony\WebpackEncoreBundle\Exception\UndefinedEntrypointException;
+
+/**
+ * Aggregate the different entry points configured in the container.
+ *
+ * Retrieve the EntrypointLookup instance from the given key.
+ *
+ * @final
+ */
+class EntrypointLookupCollection
+{
+    private $buildEntryPoints;
+
+    public function __construct(array $buildEntryPoints)
+    {
+        $this->buildEntryPoints = array_map(function ($path) {
+            return new EntrypointLookup($path);
+        }, $buildEntryPoints);
```

Comment:
Hmm. Let's tweak this slightly:

1) Make this class receive a `ContainerInterface`, which will be a service locator (like we use in the Twig extension) containing all of the `EntrypointLookup` objects.

2) In `WebpackEncoreExtension`, for each "build", register a new service dynamically. Then, create a container/service locator for all of the builds, and pass that here.

That will allow all `EntrypointLookup` objects to be managed by the container, but also passed lazily to this class, so that we're not instantiating ALL of the `EntrypointLookup` objects just to use one of them.

Reply:
I've just pushed the changes.